crowd_api/__init__.py

```diff
@@ -59,10 +59,13 @@ def get_user(self, **kwargs):
   def get_user_groups(self, **kwargs):
     groups = []
 
+    if "max_results" not in kwargs:
+      max_results = 1000
```

Comment:
you are setting `max_results` to 1000 but then you are not actually using it anywhere?

Can you change it to something like this maybe? You can do it for all the other endpoints too..

Compatible with both Python 2/3
```
"/user/group/direct?username={}&max-results={}".format(kwargs['username'], kwargs.get('max_results', 1000))
```

Reply:
whoops. For now I fixed it by setting the default to 1000 using 

```
    if "max_results" not in kwargs:
      kwargs['max_results'] = 1000
```

The other change using format() will be implemented in another PR